api/pom.xml

```diff
@@ -31,9 +31,39 @@
             <archive>users@mvc-spec.java.net</archive>
         </mailingList>
     </mailingLists>
+
+    <properties>
+        <api_package>javax.mvc</api_package>
+        <spec_version>1.0</spec_version>
+        <spec_impl_version>1.0-SNAPSHOT</spec_impl_version>
```

Comment:
Yes we can use it, but I just want to say spec-version-maven-plugin doesn't allow "1.0.0-m03-SNAPSHOT" like versioning